qemu/tests/time_conv.py

```diff
@@ -0,0 +1,44 @@
+import os
+import logging
+import time
+
+from avocado.utils import process
+
+from virttest import data_dir
+from virttest import storage
+
+
+def run(test, params, env):
+    """
+    Measure qemu-img convert time
+    1. Create a image with 1GiB.
+    2. Create a badly fragmented file with qemu-img bench.
+    3. Measure qemu-img convert time
+
+    :param test: VT test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+    image_stg = params["images"]
+    root_dir = data_dir.get_data_dir()
+    image_stg_name = storage.get_image_filename(params.object_params(image_stg),
+                                                root_dir)
+    timeout = float(params.get("timeout", 1800))
+    qemu_img_bench_cmd = params["qemu_img_bench_cmd"]
+    image_format = params["image_format"]
+    process.run(qemu_img_bench_cmd % (image_format, image_stg_name),
+                timeout=timeout, shell=True)
+    time_list = []
+    qemu_img_conv_cmd = params["qemu_img_conv_cmd"]
+    conv_img = os.path.join(os.path.dirname(image_stg_name), "convert.img")
+    for i in range(5):
+        start_time = time.time()
+        conv_cmd = process.run(qemu_img_conv_cmd % (image_format, image_format, image_stg_name, conv_img))
+        time_conv = time.time() - start_time
+        time_list.append(time_conv)
+        process.run("rm -f %s" % conv_img)
+    logging.info("The time list is: %s" % time_list)
+    max_time = params["max_time"]
+    time_list = [_ for _ in time_list if float(_) > 748.0]
```

Comment:
This is the fail list,  to distinguish with the `time_list` with L38, suggest change a name.